### ISSUE TITLE
Bruh just "May or may not be a python spam bot"

### ISSUE THREAD

**Issue by leonitousconforti** [Waffleer/Email-Server#1]
Sorry I was getting a kick out of the description you wrote. I need to send emails from a backend application through gmail, can I use this do to that?

**Comment by Waffleer** [comment]
Dont worry about it

**Comment by Waffleer** [comment]
Yep, it sends though a smtp server, what ever that means, its not gmail exclusive and works surprisingly well. Its not super complicated but it you need help I can.

**Comment by leonitousconforti** [comment]
Is there a way to use python with node.js?

**Comment by Waffleer** [comment]
Probably, not sure how to. Python is such a popular language that im sure that someone implemented a system of doing that,
